SmartPlaces.Facilities/lib/IngestionManager/src/AzureDigitalTwins/AzureDigitalTwinsGraphManager.cs

```diff
@@ -155,6 +192,12 @@ private async Task CommitCache(List<Task> cacheTasks)
             Logger.LogInformation("Completing cache commit.");
         }
 
+        /// <summary>
+        /// Asynchronously import a set of relationships into the target ADT graph.
+        /// </summary>
+        /// <param name="relationships">Relationships to upload (map key is relationship ID).</param>
+        /// <param name="retryAttempt">Retry counter.</param>
+        /// <returns>An awaitable task.</returns>
```

Comment:
Any reason this method is redefining its documentation whereas `GetModedlAsync` and `UploadGraphAsync` are using `\\\<inheritdoc/>`

Reply:
Same question for `ImportTwinAsync`

Reply:
Those other methods are defined in IOutputGraphManager and are implemented here; so they have inheritable docs that can be reused, whereas this method and ImportTwinAsync are only defined here, and thus their docs cannot be inherited.